exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -96,14 +93,14 @@ def build_and_copy_documentation(build_dir, worktree, source_branch, source_dir,
     output_dir.mkdir(parents=True)
     print(f"Copying HTML output {build_dir} to the output directory {output_dir}")
     for obj in os.listdir(build_dir):
-        shutil.move(build_dir + "/" + str(obj), output_dir)
-    # TODO test correctness of : find "build_dir" -mindepth 1 -maxdepth 1 -exec mv -t "$OUTPUT_DIR" -- {} +
+        if ".doctree" not in str(obj):
```

Comment:
Hm, maybe a better approach would be to choose a custom cache_dir with [this option](http://www.sphinx-doc.org/en/stable/man/sphinx-build.html#cmdoption-sphinx-build-d) which is not located in the build dir.
My fear is that a newer sphinx version might rename this directory .doctree => {something else} and the filter would stop working.

Reply:
That is true. I changed it.